### PR TITLE
CE-236 First iteration of action repo documentation

### PR DESCRIPTION
**Pull request by BenjaminWalkerBond**

### CODE REVIEW

File: .github/workflows/update-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# This workflow file will trigger whenever content.md or header.md has changed. It will then run the doc-gen.ps1 script.
+name: Update Docs
+on:
+  push:
+    paths:
+      - content.md
+      - header.md
```
Comment (RobCannon) [inline .github/workflows/update-docs.yaml:7]: This also needs to run when action.yaml is changed.

File: .github/workflows/update-docs.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  update-docs:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run doc-gen.ps1
+        run: ./doc-gen.ps1
+        shell: pwsh
+
```
Comment (RobCannon) [inline .github/workflows/update-docs.yaml:16]: The complicated part here is now you have to commit the changes to the repo.  Branch protection will make that harder and it probably becomes something that only runs on a pull request and it updates the PR, if necessary. 

I was only for a manual process at first, but if you want to try the full automation, then you could create an action that could be called from a PR triggered workflow.  I would start with just a workflow in this repo until you get all of the issues worked about before you create an action for this.

**Review by RobCannon — CHANGES_REQUESTED** [review]

File: .github/workflows/update-docs.yaml (hunk 1 above)
Comment (BenjaminWalkerBond) [reply .github/workflows/update-docs.yaml:6]: Added.

File: .github/workflows/update-docs.yaml (hunk 2 above)
Comment (BenjaminWalkerBond) [reply .github/workflows/update-docs.yaml:36]: That sounds like a good plan. I will get this workflow running then create an action repo afterwards.

**Review by RobCannon — APPROVED** [review]

**Pull request merged**